core/drivers/crypto/caam/include/caam_jr.h

```diff
@@ -31,6 +31,34 @@ struct caam_jrcfg {
 	uint8_t nb_jobs; /* Number of Jobs to managed */
 };
 
+/*
+ * The CAAM physical address is decorrelated from the CPU addressing mode.
+ * CAAM can manage 32 or 64 bits address depending on its version and the
+ * device.
+ */
+/*
+ * Definition of input and output ring object
+ */
+#ifdef CFG_CAAM_64BIT
+struct caam_inring_entry {
+	uint64_t desc; /* Physical address of the descriptor */
+};
+
+struct __packed caam_outring_entry {
```

Comment:
I'd prefer having the `__packed` attribute after the closing `}`, but before the `;` just as in the Linux kernel.

Reply:
Alright!